### PR TITLE
Fix #168, PSP coverage unit test

### LINKED ISSUE #168: Create PSP unit test framework and example
**Is your feature request related to a problem? Please describe.**
Need to provide example of PSP unit tests.  Coverage and functional testing are required but require a different approach.  This issue will focus on the coverage test portion.

**Describe the solution you'd like**
Provide a framework and example to perform PSP coverage test.

**Additional context**
Using MCP750 as a proof-of-concept.

**Requester Info**
Joseph Hickey, Vantage Systems, Inc.

### PR DESCRIPTION
**Pull request by jphickey**
**Describe the contribution**
Add a framework and example for coverage test of the PSP code.

Fix #168

**Testing performed**
Build and run unit test
Build and sanity check CFE (no changes to FSW)

**Expected behavior changes**
New PSP coverage test executable is built, and several files within PSP are targeted.

**System(s) tested on**
Ubuntu 20.04

**Contributor Info - All information REQUIRED for consideration of pull request**
Joseph Hickey, Vantage Systems, Inc.

### CODE REVIEW

**Comment by jphickey** [comment]
Only commit 892880a is relevant.  Will rebase after the next baseline.

The following mcp750-vxworks PSP files are currently covered as an example:
`fsw/mcp750-vxworks/src/cfe_psp_support.c`
`fsw/mcp750-vxworks/src/cfe_psp_start.c`
`fsw/shared/src/cfe_psp_exceptionstorage.c`

File: unit-test-coverage/mcp750-vxworks/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    ut_psp_libc_stubs
+    ut_osapi_stubs
+    ut_assert
+)
+
+add_test(coverage-${CFE_PSP_TARGETNAME} coverage-${CFE_PSP_TARGETNAME}-testrunner)
+
+foreach(TGT ${INSTALL_TARGET_LIST})
+    install(TARGETS coverage-${CFE_PSP_TARGETNAME}-testrunner DESTINATION ${TGTNAME}/${UT_INSTALL_SUBDIR})
+endforeach()
```
Comment (skliper) [inline unit-test-coverage/mcp750-vxworks/CMakeLists.txt:62]: `TGT` not used in install line, why foreach?

**Comment by skliper** [comment]
Also related to #14

**Comment by skliper** [comment]
What's the status on this, can it be reviewed today?

**Comment by jphickey** [comment]
Rebased to "main" -- yes this can be reviewed today.

**Comment by astrogeco** [comment]
CCB 2020-07-29 APPROVED

**Comment by astrogeco** [comment]
@skliper do you want to add this to Bootes or push to Caelum?

**Comment by skliper** [comment]
If you can merge it today, Bootes.  My logic is there is a customer waiting on it and there is no impact to flight code.  If anyone ever needs to certify Bootes it's a very valuable framework to have.

**Comment by astrogeco** [comment]
We're on the same page. Done.

**Pull request merged**